todoApp/src/App.tsx

```diff
@@ -1,15 +1,51 @@
-import React from "react";
+import React, { useState } from "react";
 import styles from "./App.module.css";
 import Button from "./components/Button";
-import Date from "./components/Date";
-import TodoLists from "./components/TodoLists";
+import DateContainer from "./components/DateContainer";
+import DisplayTodo from "./components/DisplayTodo";
+
+export interface TodoListInterface {
+  id: string;
+  title: string;
+  date: string;
+  isCompleted: boolean;
+}
 
 const App = () => {
+  let [todoTasks, setTodoTasks] = useState<TodoListInterface[]>(
+    JSON.parse(localStorage.getItem("todo") || "[]")
+  );
+
+  const buttonHandler = () => {
+    const todoArray: TodoListInterface[] = JSON.parse(
+      localStorage.getItem("todo") || "[]"
+    );
+    setTodoTasks(todoArray);
+  };
+
+  const curTime = new Date();
+  todoTasks = todoTasks.filter((todo: TodoListInterface) => {
+    let curDate = `${curTime.getDate()}/${
```

Comment:
Since you have added momentjs to the project, use utility functions provided by moment for date arithmetic.
Also, I see you have incremented month instead of day. Check it and correct this if it is wrong.